dependencies.yaml

```diff
@@ -406,6 +406,12 @@ dependencies:
     - path: images/build/debian-iptables/variants.yaml
       match: "IMAGE_VERSION: 'bullseye-v((([0-9]+)\\.([0-9]+)\\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?)(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?)'"
 
+  - name: "k8s.gcr.io/build-image/distroless-iptables"
+    version: v1.0.0
```

Comment:
Nit: Why not start with v0.1.0 or something until we've proved the stability of this?

Reply:
I trust my code Rob, no tests needed :P

Just kidding, I agree with you, fixed here